client/src/components/Header/Header.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { FaApple } from "react-icons/fa";
+import { AiOutlineSearch } from "react-icons/ai";
+import { IoBagOutline } from "react-icons/io5";
+import "../../styles/Header/Header.scss";
+function Header() {
+  const [show, setShow] = useState("");
+  const handleClick = () => {
+    console.log("clicked");
+    if (show === "") setShow("show");
+    else setShow("");
+  };
```

Comment:
Make the initial state as `null`, as that's what will be sent to the `className` and it will not be adding the empty attribute `class`.